### PR TITLE
PLATML 5951 - Incorporate sandbox related logic into R sample

### PR DESCRIPTION
**Pull request by grujicaleksandar**

### CODE REVIEW

File: recipes/R/Retail - GradientBoosting/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # from Adobe.
 #####################################################################
 
-FROM adobe/acp-dsw-ml-runtime-r:0.22.0
+FROM runtime-r-platform-sdk:1.0.0
```
Comment (grujicaleksandar) [inline recipes/R/Retail - GradientBoosting/Dockerfile]: My local ml-runtime-r, will replace it with real version when I deploy ml-runtime-r to public Docker Hub before commiting this PR.

File: recipes/R/Retail - GradientBoosting/R/applicationEvaluator.R [hunk 2]
```diff
@@ -55,7 +57,7 @@ applicationEvaluator <- setRefClass("applicationEvaluator",
       if(!is.null(tenantId)){
         if(any(names(df) == '_id')) {
           #Drop id, eventType, timestamp and rename columns
-          df <- df[,-c(1,2,3)]
+          df <- df[,-c(12,13,14)]
```
Comment (grujicaleksandar) [inline recipes/R/Retail - GradientBoosting/R/applicationEvaluator.R:58]: _id, eventType and timestamp are no longer at places 1,2,3.
Now they are on 12,13 and 14.

File: recipes/R/Retail - GradientBoosting/R/applicationScorer.R [hunk 3]
```diff
@@ -64,7 +66,7 @@ applicationScorer <- setRefClass("applicationScorer",
       if(!is.null(tenantId)){
         if(any(names(df) == '_id')) {
           #Drop id, eventType, timestamp and rename columns
-          df <- df[,-c(1,2,3)]
+          df <- df[,-c(12,13,14)]
```
Comment (grujicaleksandar) [inline recipes/R/Retail - GradientBoosting/R/applicationScorer.R:67]: _id, eventType and timestamp are no longer at places 1,2,3.
Now they are on 12,13 and 14.

**Comment by tumulurik** [comment]
The changes LGTM, ready to approve once the SDK version is updated and testing is complete.

File: recipes/R/Retail - GradientBoosting/Dockerfile [hunk 4]
```diff
@@ -15,7 +15,7 @@
 # from Adobe.
 #####################################################################
 
-FROM adobe/acp-dsw-ml-runtime-r:0.22.0
+FROM adobe/acp-dsw-ml-runtime-r:0.32.0
```
Comment (grujicaleksandar) [inline recipes/R/Retail - GradientBoosting/Dockerfile]: This need to be changed with the real version once ml-runtime-r is pushed to public Docker Hub during release job on Jenkins.
PR for ml-runtime-r : https://git.corp.adobe.com/ml/ml-runtime-r/pull/57

**Review by NemanjaL94 — APPROVED** [review]

**Review by tumulurik — APPROVED** [review]

**Pull request merged**